### DISCUSSION TITLE
[v5] Using 2 datasources with Lucid

### DISCUSSION THREAD

**Discussion by guntribam** [adonisjs/core#2496] [Help] [answered]
Can I use 2 datasources with Lucid? For example: I have 2 databases in sql server , one containing corporate user information and such, and in the other database my app tables. Can Lucid abstract the 2 databases as if they were one and map tables from both?

**Comment by artem-a** [comment, marked as answer]
@guntribam Hi yes you can. 
Setting connection for the model
https://preview.adonisjs.com/guides/models/introduction#models-config

Switch between connections
https://preview.adonisjs.com/guides/database/connections-management#switching-between-connections
